COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md

```diff
@@ -1370,181 +1398,191 @@ In this lab exercise you will connect to an IBM Z system, view a simple COBOL he
 
 4. A box appears to define a new profile.  Click + to the left of Create a New Connection to z/OS as shown in Figure  5.
 
-![](Images/image050.jpg)
+![](Images/image050.png)
 
 *Figure  5.  Create a new connection to z/OS*
 
- 
+
 5. Select a name to enter, then enter.  Figure  6. used `LearnCOBOL` as the selected connection name.
 
-![](Images/image053.jpg)
+![](Images/image053.png)
 
 *Figure  6.  Set connection name*
 
- 
+
 6. VSCode prompts for z/OSMF URL and port as shown in Figure  7.  The z/OSMF URL and port will normally be provided by z/OS System Administrator.
 
-![](Images/image056.jpg)
+![](Images/image056.png)
 
 *Figure  7.  z/OSMF URL*
 
 7. A sample z/OSMF URL and port is entered as shown in Figure  8.
 
-![](Images/image058.jpg)
+![](Images/image058.png)
 
 *Figure  8.  Specified z/OSMF URL*
 
 8. The connection prompts for Username as shown in Figure  9.
 
-![](Images/image061.jpg)
+![](Images/image061.png)
 
 *Figure  9.  User name prompt*
 
 9. **Please enter the username assigned to you! Do not use the sample username of Z99998.** A sample username, is entered as shown in Figure  10.  The ID is assigned by the System Administrator.
 
-![](Images/image064.jpg)
+![](Images/image064.png)
 
 *Figure  10.  Specified user name*
 
 
 10. The connection prompts for the password as shown in Figure  11.
 
-![](Images/image067.jpg)
+![](Images/image067.png)
 
 *Figure  11.  Password prompt*
 
 11. Enter the password as shown in Figure  12.
 
-![](Images/image070.jpg)
+![](Images/image070.png)
 
 *Figure  12.  Specified password*
 
 12. Select **False - Accept connections with self-signed certificates** to authorize workstation connection as shown in Figure  13.
 
 ![](Images/image073.png)
 
+13. Press **Enter** for the rest of the prompts to choose the default values
+
+![](Images/image073-1.png)
+
+![](Images/image073-2.png)
+
+![](Images/image073-3.png)
+
+![](Images/image073-4.png)
+
 *Figure  13.  Accept connections with self-signed certifications*
 
 13. Result is Favorites in the Data Sets, Unix System Services, and Jobs sections as shown in Figure  14.
 
-![](Images/image078.jpg)
+![](Images/image078.png)
 
 *Figure  14.  Favorites*
 
 14.  Again, click on the + to the far right on the Data Sets selection.  Result is another prompt to Create a New Connection to z/OS.  LearnCOBOL is in the connection list.  Select **LearnCOBOL** for the Data Sets available to the previously defined LearnCOBOL connection to z/OS as shown in Figure  15.
 
-![](Images/image080.jpg)
+![](Images/image080.png)
 
 *Figure  15.  LearnCOBOL connection*
```

Comment:
I think with the latest Zowe Explorer, once the profile is created, it's automatically added to the list, so this step will no longer be needed.

Reply:
yes you are right, should I replace it with something like "your profile will be added automatically to the DATASET list"
and add a note that "you can click on the + to the far right on the Data Sets selection to see all your connections that are not in the List and be able to add them to the list"

Reply:
Hmm, that would be nice I think...